.gitignore

```diff
@@ -15,3 +15,5 @@ node_modules
 .DS_Store
 
 coverage
+
+*.sublime-*
```

Comment:
Good catch. I normally have this i my sublime config, but I'm still getting setup on my new machine and havent copied over my preferences. I'll also have to catch up with you on the lob eslint config. When I run it I get about 100 linter errors that were there before I edited the code.